react/BankInvoiceSection.tsx

```diff
@@ -50,10 +59,10 @@ const BankInvoiceSection: FC = () => {
           values={{ paymentSystemName }}
         />
       </header>
-      <div className="flex-l justify-between items-center mt6">
+      <div className={`${handles.barCodeFlexContainer} flex-l justify-between items-center mt6`}>
         {barCodeNumber && <BarCode barCodeNumber={barCodeNumber} />}
         {isURLValid && (
-          <div className={`mt5 mt0-l ${barCodeNumber ? 'ml5-l' : ''}`}>
+          <div className={`${handles.printBilletButtonWrapper} mt5 mt0-l ${barCodeNumber ? 'ml5-l' : ''}`}>
```

Comment:
This handle was not added to the readme's list! Couldn't it be called just `printButtonWrapper`?

Reply:
Indeed I forgot.